### DISCUSSION TITLE
Can't fix error " invalid Link with a child " with npx @next/codemod new-link .

### DISCUSSION THREAD

**Discussion by AdamGaoua** [vercel/next.js#45735] [Help]
I start a next app project. I used this code : 

```
<Link href={href} legacyBehavior>
      <a
        className={`nav__item ${
          active ? "active" : ""
        }`}
      >
```

My terminal sent me this error : 
`Error: Invalid <Link> with <a> child. Please remove <a> or use <Link legacyBehavior>.`

So I tried this : npx @next/codemod new-link . from https://nextjs.org/docs/messages/invalid-new-link-with-extra-anchor

After, my terminal sent me : Thank you for using @next/codemod!
But before we continue, please stash or commit your git changes.

Event after git commit -m "message", I keep the first error message : `Error: Invalid <Link> with <a> child. Please remove <a> or use <Link legacyBehavior>.`

**Comment by icyJoseph** [comment]
Have you double checked that the codemod fixed the Link issue? Like, in the code itself? 

Are you perhaps using a library or something that the codemod can't fix?

**Reply by icyJoseph** [reply]
@AdamGaoua but like the command said:

```console
Thank you for using @next/codemod!
But before we continue, please stash or commit your git changes.
```

So you have to commit or stash and run the tool again, because it doesn't seem like it did anything.

Did you run it again after having a clean git status?

Are there anymore Links in the app?

**Comment by AdamGaoua** [comment]
My error is : "Error: Invalid <Link> with <a> child."

From the official doc (https://nextjs.org/docs/messages/invalid-new-link-with-extra-anchor), codemod fix this problem. I don't use any library.
